src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -39,18 +40,19 @@ export class PastAppointmentsList extends React.Component {
     }
   }
 
-  componentDidUpdate(prevProps) {
-    if (
-      prevProps.appointments.pastStatus === FETCH_STATUS.loading &&
-      this.props.appointments.pastStatus === FETCH_STATUS.succeeded
-    ) {
-      focusElement('#queryResultLabel');
-    }
-  }
+  // componentDidUpdate(prevProps) {
+  //   if (
+  //     prevProps.appointments.pastStatus === FETCH_STATUS.loading &&
+  //     this.props.appointments.pastStatus === FETCH_STATUS.succeeded
+  //   ) {
+  //     focusElement('#queryResultLabel');
+  //   }
+  // }
```

Comment:
I will remove commented code if you agree that 'componentDidUpdate' is no longer needed base on comment from Trevor.

Reply:
No, I think we still need it to do this: "We will only move focus to the “Showing appointments…” when users press the Update button when using the filters dropdown."

I think we just need to do the thing we talked about earlier, adding `!this.state.isInitialMount` to the condition.